src/Sarif.Sarifer/TestData/proof-of-concept.sarif

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SpellChecker",
```

Comment:
>SpellChecker [](start = 19, length = 12)

Can you please embed Example.cs into this log file as text? Using the 'artifacts' table. Add an artifact index as well, of course.

Reply:
Sure.

---
In reply to: [514515148](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514515148) [](ancestors = 514515148)